Code/EnginePlugins/AngelScriptPlugin/Runtime/AsCompilation.cpp

```diff
@@ -191,8 +195,21 @@ asIScriptModule* ezAngelScriptEngineSingleton::CompileModule(ezStringView sModul
   if (asPP.Process(fullCode).Failed())
   {
     ezLog::Error("Failed to pre-process AngelScript");
```

Comment:
If there is an ezResult return type, then the function shouldn't also log an error. Instead the caller should log on failure.

Reply:
Good point! Fixed.